apps/fontkorean/metadata.json

```diff
@@ -0,0 +1,16 @@
+{ "id": "fontkorean",
+  "name": "Korean font",
+  "version":"0.01",
+  "description": "Installs a font data, Unifont characters for Korean **Requires 420 KB storage**",
+  "icon": "fontkorean_icon.png",
+  "tags": "font",
+  "type": "module",
+  "provides_modules" : ["font"],
+  "supports" : ["BANGLEJS2"],
+  "readme": "README.md",
+  "storage": [
+    {"name":"font","url":"lib.js"},
+    {"name":"fontkorean.boot.js","url":"boot.js"},
+    {"name":"fontkorean.pbf","url":"fontkorean.pbf"}

```

Comment:
This is nit picky - sorry. I suggest you rename the file in the folder to just `font.pbf`. But like you do name it `fontkorean.pbf` when uploading to the watch.

Reply:
so: `{"name":"fontkorean.pbf","url":"font.pbf"}`

Reply:
That makes sense. I agree. I will make the change.

Reply:
Shouldn't it be `{"name":"font.pbf","url":"fontkorean.pbf"}`

Reply:
No, the url points to the file location on the repository and the "name" decides what the name is when loaded to the Bangle.

For `fontall` it looks like this: `{"name":"fontall.pbf","url":"font.pbf"}`

Reply:
Ah.. ok. Thanks.